client/scss/vendor/jquery.Jcrop.min.css

```diff
@@ -1,6 +1,6 @@
 /* jquery.Jcrop.min.css v0.9.12 (build:20130126) */
 .jcrop-holder{direction:ltr;text-align:left;}
-.jcrop-vline,.jcrop-hline{background:#FFF url(../../wagtailimages/images/Jcrop.gif);font-size:0;position:absolute;}
+.jcrop-vline,.jcrop-hline{background:#FFF;font-size:0;position:absolute;}
```

Comment:
I'm not sure if this is fixable, maybe it's part of https://github.com/web-infra-dev/rspack/issues/5340